### DISCUSSION TITLE
Foreign key not added by default (for example in M2O relations)?

### DISCUSSION THREAD

**Discussion by gremo** [directus/directus#5337] [Archive]
Just a very newbie question about Directus. Adding a relation, say M2O (products -> category) doesn't add a foreign key between `product.category` field and` category.id field`.

Is this something "by design" (am I supposed to do it using migrations and/or direct database management) or there is something wrong with my setup? 

Thanks!

**Comment by rijkvanzanten** [comment]
This is going to be added very soon 🙂 

https://github.com/directus/directus/issues/3664

**Reply by gremo** [reply]
Really nice 👍 ! How much "soon", do we know?

**Reply by rijkvanzanten** [reply]
It's on the required-before-we-can-launch-v9 list, and there's only a handful more things on that list: https://github.com/directus/directus/milestone/24 so without making any hard deadline promises, it's probably somewhere in the next week or two 🙂
